### ISSUE TITLE
Release recent changes to pypi?

### ISSUE THREAD

**Issue by bac** [sk-/git-lint#157]
Is a release to pypi with the recent changes you made forthcoming? Would like to easily take advantage of them.  Thanks.

**Comment by varac** [comment]
Yes please! Last pypi release [0.1.2](https://pypi.org/project/git-lint/0.1.2/) is nearly a year old.
Also, please keep github releases/tags and pypi versions in sync, the only [github release](https://github.com/sk-/git-lint/releases) is 0.0.7 and from 2015.
